e2e-tests/portal/cypress/e2e/journeys/maker/abandon-a-deal.spec.js

```diff
@@ -2,18 +2,16 @@ const { contract, contractDelete, defaults } = require('../../pages');
 const { successMessage } = require('../../partials');
 const relative = require('../../relativeURL');
 const MOCK_USERS = require('../../../../../e2e-fixtures');
-const twentyOneDeals = require('../../../fixtures/deal-dashboard-data');
 
 const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
 
 context('A maker selects to abandon a contract from the view-contract page', () => {
   let deal;
 
   before(() => {
-    const aDealInStatus = (status) => twentyOneDeals.filter((aDeal) => status === aDeal.status)[0];
-
     cy.deleteDeals(ADMIN);
-    cy.insertOneDeal(aDealInStatus('Draft'), BANK1_MAKER1).then((insertedDeal) => {
+
+    cy.createBssDeal({ readyForCheck: true }).then((insertedDeal) => {
```

Comment:
This should be `cy.createBssDeal({});` so that the deal has a `DRAFT` status like the previous test